src/main/java/team/baymax/logic/commands/patient/ListPatientCommand.java

```diff
@@ -13,7 +13,7 @@
  */
 public class ListPatientCommand extends Command {
 
-    public static final String COMMAND_WORD = "listpatient";
+    public static final String COMMAND_WORD = "lspat";
```

Comment:
shotening to "pat" from "patient" could be confusing for the user as "pat" is not a conventional short form for patient (unlike appt). Also, standardise list or ls.